os/booting-on-google-compute-engine.md

```diff
@@ -104,6 +108,34 @@ gcloud compute ssh --zone us-central1-a core@<instance-name>
 
 Users other than `core`, which are set up by the GCE account manager, may not be a member of required groups. If you have issues, try running commands such as `journalctl` with sudo.
 
+### Using OS Login
+
+You can log in using your Google account on instances with OS Login enabled. OS Login needs to be [enabled in the GCE console](https://cloud.google.com/compute/docs/instances/managing-instance-access#enable_oslogin) and on the instance. It is enabled by default on the instance for nodes provisioned with Container Linux 1849.0.0 or later. Once enabled, you can log into your Container Linux instances using:
+
+```sh
+gcloud compute ssh --zone us-central1-a <instance-name>
+```
+
+Which will use your GCE user to log in.
+
+
+#### Disabling OS Login on newly provisioned nodes
+
+You can disable the OS Login functionality by masking the `oem-gce-enable-oslogin.service` unit:
```

Comment:
Since masking the unit only works via Ignition config, maybe e.g. "by masking the ... unit using a Container Linux Config:"

Reply:
hrmmmph. I didn't think of cloudinit. This means that cloudinit users _can't_ disable it without reboot. I dunno how that sits with me. If they're disabling for some security reason, booting once with it is not good (granted they can disable it from the gce side)

Reply:
It wouldn't be the first knob that can't be turned with coreos-cloudinit.